synapseAnnotations/data/experimentalData.json

```
@@ -325,6 +325,16 @@
         "value": "Genotyping",
         "description": "The determination of the DNA sequence of an individual.",
         "source": "http://purl.obolibrary.org/obo/NCIT_C45447"
+      },
+      {
+      	"value": "TMT",
+      	"description":"A high-throughput multiplexed proteomics technique.", 
```

```suggestion
      	"description":"An isobaric labeling technique that uses tags containing four regions with the same total molecular weights and structure, so that during chromatographic or electrophoretic separation and in single MS mode, molecules labelled with different tags are indistinguishable.", 
```

Just noticed that this description differs from the source. Can we update it?